src/add/form-data.js

```diff
@@ -12,6 +12,14 @@ exports.toFormData = async input => {
   let i = 0
 
   for await (const file of files) {
+    if (file.mtime) {
+      formData.append(`file-${i}-mtime`, file.mtime)
```

Comment:
What if this is `dir-${i}`?

Reply:
The field names aren't used when parsing multipart requests - if it's a dir we detect that from the content type.